KaiUIngInferno/src/ui/SoftKey.tsx

```diff
@@ -10,7 +10,7 @@ interface ButtonProps {
 
 const prefixCls = "kai-softkey";
 function Button(props: ButtonProps) {
-  let renderedIcon: JSX.Element;
+  let renderedIcon: HTMLElement;
```

Comment:
I don't think that is correct, I think `HTMLElement` refers to a reference to an element in the DOM, but later you set it to jsx, which is not an DOM element.